### DISCUSSION TITLE
q-input - go to next field after certain number of values

### DISCUSSION THREAD

**Discussion by CynthiaBlue** [quasarframework/quasar#13358] [CLI - Electron mode] [answered]
I would like to have a q-input that takes a value with the length of 1, then when the user types in that one value, the cursor automatically goes (tabs?) to the next field.  Is this possible, and if so how? I have been googling but I don't know if I'm googling for the right thing. 

Thanks.

**Comment by pdanpdan** [comment]
To be honest I don't know either if you are googling for the right thing

**Comment by Hashen110** [comment]
I think he is trying to implement something like this

![OTP](https://user-images.githubusercontent.com/37979557/166980389-43e3906e-4d90-4621-b743-b9042e4e9539.png)

**Reply by CynthiaBlue** [reply]
yeah kinda like that.

**Comment by metalsadman** [comment, marked as answer]
check these codepens:
https://codepen.io/tobias-mesquita/pen/QWObrVq?editors=1010 

https://codepen.io/euphemism/pen/MWOWWMX?editors=1011

**Reply by CynthiaBlue** [reply]
Awesome thanks!
